l1-contracts/src/core/Rollup.sol

```diff
@@ -952,9 +1001,10 @@ contract Rollup is EIP712("Aztec Rollup", "1"), Leonidas, IRollup, ITestRollup {
     require(timestamp <= _currentTime, Errors.Rollup__TimestampInFuture(_currentTime, timestamp));
 
     // Check if the data is available
+    // TODO(Miranda): either add blobhash to the header or find a way of validating it here
     require(
-      _flags.ignoreDA || _header.contentCommitment.txsEffectsHash == _txsEffectsHash,
-      Errors.Rollup__UnavailableTxs(_header.contentCommitment.txsEffectsHash)
+      _flags.ignoreDA || 1 == 1, // _header.blobHash == _blobHash,
```

Comment:
@ TMNT team: here we need to check the data has been published. I could add the EVM's blobHash to the block header (we can calculate it in ts in advance) and uncomment the check? 
However it could be doable without adding more fields to the header by having some `isAvailable` mapping filled once `_validateBlob` is called. Not sure which is preferable based on the team's priorities!